src/controllers/Trip.js

```diff
@@ -352,6 +353,75 @@ class TripController {
       return res.status(response.code).json(response);
     }
   }
+
+  /**
+   * @description - method to retrieve a users favourites destinations
+   * @param {object} req - HTTP request object
+   * @param {object} res - HTTP response object
+   * @returns {object} - object
+   */
+  static async getMostVisitedBranch(req, res) {
+    const { companyId } = req.payload.payload;
```

Comment:
Not that it is necessary Peter, but you can try this here also:

```
const { payload: { companyId } } = req.payload;
```
Not that it matters though👍